bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -203,20 +216,43 @@ private String getHtmlWithInjectedAttributes(String html) {
         return StringUtils.replace(updatedHtml,"<link ", "<link " + cssAttributes.toString());
     }
 
+    /**
+     * Gets the fragment for an attribute, conditioned for include
```

Comment:
I'd phrase it as: Gets the fragment for an attribute, based on its name and flag to include

Reply:
Done in 000c479